src/libs/tools/src/plugindatabase.cpp

```diff
@@ -223,6 +223,21 @@ std::string ModulesPluginDatabase::lookupInfo (PluginSpec const & spec, std::str
 	return plugin->lookupInfo (which);
 }
 
+PluginDatabase::func_t ModulesPluginDatabase::getSymbol (PluginSpec const & spec, std::string const & which) const
+{
+	PluginPtr plugin;
+	try
+	{
+		plugin = impl->modules.load (spec.getName (), spec.getConfig ());
+	}
+	catch (...)
+	{
+		throw;
+	}
```

Comment:
If you copied it from somewhere, there might be an use case I didn't think of. I mean, it is quite obvious that you see this way a lot easier that there is an exception thrown by the invoked method and that it is escalated, but I guess the compiler should warn you about the exception anyway when using the `getSymbol` function without a try-catch clause.